js/data/createDonuts.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+import * as d3 from 'd3-shape';
+
+export default function createDonuts(data, weekday){
+
+  var svg = document.createElementNS("http://www.w3.org/2000/svg", "svg");
+  svg.setAttribute('width', '600');
+  svg.setAttribute('height', '600');
+  svg.setAttributeNS("http://www.w3.org/2000/xmlns/", "xmlns:xlink", "http://www.w3.org/1999/xlink");
+
+  document.body.appendChild(svg);
+
+  var color = ["#00585F","#009393","#63A69F","#F2E1AC","#F2836B","#CD2C24","#F2594B"]
+
+  var arcs = d3.pie()(data);
+  var i = 0;
+  arcs.forEach(function(e){
```

Comment:
You don't need the `i` variable as you could have just done:
```
arcs.forEach(function(e, i){
```
The index is provided by forEach for free